sys/shell/commands/sc_gnrc_icmpv6_echo.c

```diff
@@ -338,7 +338,8 @@ static void _print_reply(_ping_data_t *data, gnrc_pktsnip_t *icmpv6,
         uint16_t recv_seq;
 
         /* not our ping */
-        if (byteorder_ntohs(icmpv6_hdr->id) != data->id) {
+        if ((byteorder_ntohs(icmpv6_hdr->id) != data->id) &&
+            !ipv6_addr_equal(from, &data->host)) {
```

Comment:
Maybe it would make sense to print a short line to indicate that potential mischief happened? E.g. the potential driver issue with either at85rf2xx or the nrf52 802.15.4 I noticed would no longer be detectable if the unexpected pongs are silently dropped.

Reply:
Mh... The original IP implementation does not seem to care about this + it could be legitimate (e.g. when two threads pinging at the same time for some reason) that a packet that holds this condition arrives here. What is the potential driver issue again?